.idea/misc.xml

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="EclipseCodeFormatter">
```

Comment:
This changes shouldn't be commited, but as .idea wasn't on the gitignore it was, sorry :)